Index: configure.in
===================================================================
RCS file: /var/cvs/libtool/libltdl/configure.in,v
retrieving revision 1.47
diff -u -u -r1.47 configure.in
--- configure.in	1999/05/20 17:54:09	1.47
+++ configure.in	1999/05/30 16:40:17
@@ -1,35 +1,12 @@
-dnl Process this file with autoconf to create configure.
 
-AC_INIT(ltdl.c)
-
-if test -z "$enable_ltdl_install$enable_ltdl_convenience"; then
-  if test -f ${srcdir}/ltconfig && test -f ${srcdir}/ltmain.sh; then
-    # if libltdl is libtoolized, it is assumed to be stand-alone and
-    # installed unless the command line overrides it (tested above)
-    enable_ltdl_install=yes
-  else
-    AC_MSG_WARN([*** The top-level configure must select either])
-    AC_MSG_WARN([*** [A""C_LIBLTDL_INSTALLABLE] or [A""C_LIBLTDL_CONVENIENCE].])
-    AC_MSG_ERROR([*** Maybe you want to --enable-ltdl-install?])
-  fi
-fi
-
-AM_INIT_AUTOMAKE(libltdl,1.0,-)
-AM_CONFIG_HEADER(config.h)
-AM_MAINTAINER_MODE
-
-AC_PROG_CC
+AC_DEFUN(AC_LIBLTDL,
+[
+AC_REQUIRE([AC_PROG_CC])
 AC_C_CONST
 AC_C_INLINE
-AM_PROG_LIBTOOL
+AC_REQUIRE([AM_PROG_LIBTOOL])
 AC_SUBST(LIBTOOL_DEPS)
 
-AC_ARG_ENABLE(ltdl-install,
-[  --enable-ltdl-install   install libltdl])
-
-AM_CONDITIONAL(INSTALL_LTDL, test x"${enable_ltdl_install-no}" != xno)
-AM_CONDITIONAL(CONVENIENCE_LTDL, test x"${enable_ltdl_convenience-no}" != xno)
-
 dnl Read the libtool configuration
 rm -f conftest
 ./libtool --config > conftest
@@ -88,11 +65,19 @@
 fi
 
 LIBADD_DL=
-AC_CHECK_LIB(dl, dlopen, [AC_DEFINE(HAVE_LIBDL, 1) LIBADD_DL="-ldl"],
-[AC_CHECK_FUNC(dlopen, [AC_DEFINE(HAVE_LIBDL, 1)])])
-AC_CHECK_FUNC(shl_load, [AC_DEFINE(HAVE_SHL_LOAD, 1)],
-[AC_CHECK_LIB(dld, shl_load, [AC_DEFINE(HAVE_SHL_LOAD, 1) LIBADD_DL="$LIBADD_DL -ldld"])])
-AC_CHECK_LIB(dld, dld_link, [AC_DEFINE(HAVE_DLD, 1)dnl
+ac_have_libdl=no
+ac_have_shl_load=no
+AC_CHECK_LIB(dl, dlopen, [ac_have_libdl=yes; LIBADD_DL="-ldl"],
+[AC_CHECK_FUNC(dlopen, [ac_have_libdl=yes])])
+if test "$ac_have_libdl" = yes; then
+  AC_DEFINE(HAVE_LIBDL, 1, [Define if you have the libdl library or equivalent.])
+fi
+AC_CHECK_FUNC(shl_load, [ac_have_shl_load=yes],
+[AC_CHECK_LIB(dld, shl_load, [ac_have_shl_load=yes; LIBADD_DL="$LIBADD_DL -ldld"])])
+if test "$ac_have_shl_load" = yes; then
+  AC_DEFINE(HAVE_SHL_LOAD, 1, [Define if you have the shl_load function.])
+fi
+AC_CHECK_LIB(dld, dld_link, [AC_DEFINE(HAVE_DLD, 1, [Define if you have the GNU dld library.])dnl
 test "x$ac_cv_lib_dld_shl_load" = yes || LIBADD_DL="$LIBADD_DL -ldld"])
 AC_SUBST(LIBADD_DL)
 
@@ -364,8 +349,7 @@
     [Define if dlsym() requires a leading underscode in symbol names. ])
 fi
 
-dnl Output the makefile
-AC_OUTPUT(Makefile)
+])
 
 # Local Variables:
 # mode:shell-script
