diff -Nru kdelibs.old/Doxyfile kdelibs.new/Doxyfile
--- kdelibs.old/Doxyfile	Thu May  9 23:53:48 2002
+++ kdelibs.new/Doxyfile	Sat May 18 03:22:31 2002
@@ -119,7 +119,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
 #---------------------------------------------------------------------------
-GENERATE_LATEX         = YES
+GENERATE_LATEX         = NO
 LATEX_OUTPUT           = 
 COMPACT_LATEX          = NO
 PAPER_TYPE             = a4wide
@@ -140,7 +140,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the man page output
 #---------------------------------------------------------------------------
-GENERATE_MAN           = YES
+GENERATE_MAN           = NO
 MAN_OUTPUT             = man
 MAN_EXTENSION          = .kde3
 MAN_LINKS              = YES
