diff -ru kdelibs.old/kio/http/kcookiejar/kcookiewin.cpp kdelibs.new/kio/http/kcookiejar/kcookiewin.cpp
--- kdelibs.old/kio/http/kcookiejar/kcookiewin.cpp	Sun May 13 18:36:48 2001
+++ kdelibs.new/kio/http/kcookiejar/kcookiewin.cpp	Sun Jan 27 22:22:58 2002
@@ -153,10 +153,10 @@
     QBoxLayout* bbLay = new QHBoxLayout( bbox );
     bbLay->setSpacing( KDialog::spacingHint() );
     QPushButton* btn = new QPushButton( i18n("&Accept"), bbox );
-    btn->setDefault( true );
     connect( btn, SIGNAL(clicked()), SLOT(accept()) );
     bbLay->addWidget( btn );
     btn = new QPushButton( i18n("&Reject"), bbox );
+    btn->setDefault( true );
     connect( btn, SIGNAL(clicked()), SLOT(reject()) );
     bbLay->addWidget( btn );
     bbLay->addStretch( 1 );    
