--- kdelibs.old/kdecore/kstandarddirs.cpp	Wed Jan 21 06:59:44 2004
+++ kdelibs/kdecore/kstandarddirs.cpp	Wed Jan 21 07:00:05 2004
@@ -849,7 +849,7 @@
     if (!strcmp(type, "data"))
 	return "share/apps/";
     if (!strcmp(type, "html"))
-	return "share/doc/HTML/";
+	return "share/doc/kde/HTML/";
     if (!strcmp(type, "icon"))
 	return "share/icons/";
     if (!strcmp(type, "config"))
@@ -869,7 +869,7 @@
     if (!strcmp(type, "mime"))
 	return "share/mimelnk/";
     if (!strcmp(type, "cgi"))
-	return "cgi-bin/";
+	return "lib/cgi-bin/";
     if (!strcmp(type, "wallpaper"))
 	return "share/wallpapers/";
     if (!strcmp(type, "templates"))
