diff -b -u kdelibs.old/khtml/java/Makefile.am kdelibs.new/khtml/java/Makefile.am
--- kdelibs.old/khtml/java/Makefile.am	22 May 2002 00:29:33
+++ kdelibs.new/khtml/java/Makefile.am	24 May 2002 23:00:00
@@ -12,7 +12,7 @@
 include_HEADERS = kjavaapplet.h kjavaappletcontext.h \
 	kjavaappletserver.h kjavaprocess.h
 
-libkjava_la_LDFLAGS = $(KDE_MT_LDFLAGS) -version-info 1:0 -no-undefined
+libkjava_la_LDFLAGS = $(KDE_MT_LDFLAGS) -version-info 2:0 -no-undefined
 libkjava_la_LIBADD = $(LIB_KPARTS)
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/khtml $(all_includes) 
diff -Nur kdelibs.old/kjs/Makefile.am kdelibs.new/kjs/Makefile.am
--- kdelibs.old/kjs/Makefile.am	28 May 2002 17:38:00
+++ kdelibs/kjs/Makefile.am	28 May 2002 17:57:00
@@ -40,7 +40,7 @@
 		 math_object.h object_object.h regexp_object.h error_object.h \
 		 function.h debugger.h property_map.h
 
-libkjs_la_LDFLAGS = -version-info 1:0 -no-undefined $(USER_LDFLAGS) $(all_libraries)
+libkjs_la_LDFLAGS = -version-info 2:0 -no-undefined $(USER_LDFLAGS) $(all_libraries)
 libkjs_la_LIBADD = -lm $(LIBPCRE)
 
 EXTRA_DIST = grammar.y
diff -Nur kdelibs.old/interfaces/ktexteditor/Makefile.am kdelibs.new/interfaces/ktexteditor/Makefile.am
--- kdelibs.old/interfaces/ktexteditor/Makefile.am	28 May 2002 17:37:58
+++ kdelibs/interfaces/ktexteditor/Makefile.am	28 May 2002 17:57:14
@@ -14,7 +14,7 @@
     configinterfaceextension.cpp encodinginterface.cpp sessionconfiginterface.cpp \
     viewstatusmsginterface.cpp
     
-libktexteditor_la_LDFLAGS = $(all_libraries) $(KDE_RPATH)
+libktexteditor_la_LDFLAGS = -version-info 1:0 $(all_libraries) $(KDE_RPATH)
 libktexteditor_la_LIBADD = $(LIB_KPARTS) $(top_builddir)/dcop/libDCOP.la
 
 ktexteditorinclude_HEADERS = document.h view.h editor.h plugin.h editinterface.h undointerface.h \
