diff -ur kdelibs.old/kdecore/ksocks.cpp kdelibs.new/kdecore/ksocks.cpp
--- kdelibs.old/kdecore/ksocks.cpp	Sat Nov  3 01:33:27 2001
+++ kdelibs.new/kdecore/ksocks.cpp	Sun Jan 27 23:12:19 2002
@@ -216,7 +216,8 @@
              << "/usr/local/lib/"
              << "/usr/local/socks5/lib/"
              << "/opt/socks5/lib/";
-   _libNames << "libsocks.so"                  // Dante
+   _libNames << "libsocksd.so.0"                  // Dante
+             << "libsocks.so.4"
              << "libsocks5.so"                 // ?
              << "libsocks5_sh.so";             // NEC
 
diff -ur kdelibs.old/kssl/kopenssl.cc kdelibs.new/kssl/kopenssl.cc
--- kdelibs.old/kssl/kopenssl.cc	Tue Aug 21 11:07:31 2001
+++ kdelibs.new/kssl/kopenssl.cc	Sun Jan 27 23:12:19 2002
@@ -188,20 +188,12 @@
    }
 #else
    libpaths << "/usr/lib/"
-            << "/usr/local/lib/"
-            << "/usr/local/openssl/lib/"
-            << "/usr/local/ssl/lib/"
-            << "/opt/openssl/lib/"
             << "";
 
 // FIXME: #define here for the various OS types to optimize
-   libnamess << "libssl.so.0"
-             << "libssl.so"
-             << "libssl.sl";
+   libnamess << "libssl.so.0.9.6";
 
-   libnamesc << "libcrypto.so.0"
-             << "libcrypto.so"
-             << "libcrypto.sl";
+   libnamesc << "libcrypto.so.0.9.6";
 
    for (QStringList::Iterator it = libpaths.begin();
                               it != libpaths.end();
