diff -Nru kdelibs.old/Doxyfile kdelibs.new/Doxyfile
--- kdelibs.old/Doxyfile	Thu Mar 21 20:25:17 2002
+++ kdelibs.new/Doxyfile	Sat Apr  6 02:36:14 2002
@@ -120,7 +120,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
 #---------------------------------------------------------------------------
-GENERATE_LATEX         = YES
+GENERATE_LATEX         = NO
 LATEX_OUTPUT           = 
 COMPACT_LATEX          = NO
 PAPER_TYPE             = a4wide
@@ -141,7 +141,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the man page output
 #---------------------------------------------------------------------------
-GENERATE_MAN           = YES
+GENERATE_MAN           = NO
 MAN_OUTPUT             = man
 MAN_EXTENSION          = .kde3
 MAN_LINKS              = YES
