diff -Nru kdelibs.old/Doxyfile kdelibs.new/Doxyfile
--- kdelibs.old/Doxyfile	Wed Jan  9 09:53:13 2002
+++ kdelibs.new/Doxyfile	Tue Mar 12 16:16:03 2002
@@ -120,7 +120,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the LaTeX output
 #---------------------------------------------------------------------------
-GENERATE_LATEX         = YES
+GENERATE_LATEX         = NO
 LATEX_OUTPUT           = 
 COMPACT_LATEX          = NO
 PAPER_TYPE             = a4wide
@@ -141,7 +141,7 @@
 #---------------------------------------------------------------------------
 # configuration options related to the man page output
 #---------------------------------------------------------------------------
-GENERATE_MAN           = YES
+GENERATE_MAN           = NO
 MAN_OUTPUT             = man
 MAN_EXTENSION          = .kde3
 MAN_LINKS              = YES
@@ -172,7 +172,7 @@
 # Configuration options related to the dot tool   
 #---------------------------------------------------------------------------
 CLASS_DIAGRAMS         = YES
-HAVE_DOT               = YES
+HAVE_DOT               = NO
 CLASS_GRAPH            = YES
 COLLABORATION_GRAPH    = YES
 TEMPLATE_RELATIONS     = YES
