diff -ur kdelibs.old/kdecore/kapp.cpp kdelibs.new/kdecore/kapp.cpp
--- kdelibs.old/kdecore/kapp.cpp	Tue Dec  4 19:42:24 2001
+++ kdelibs.new/kdecore/kapp.cpp	Sun Jan 27 22:11:44 2002
@@ -1808,7 +1808,7 @@
    // has to be executed by the terminal has to be supplied (e.g. something
    // like '/opt/kde2/bin/konsole -e %p'). - Frerich
    if (config.readBoolEntry("TerminalClient", false))
-      command = "konsole -e " + command;
+      command = "x-terminal-emulator -e " + command;
 
    QString address = KURL::decode_string(mailtoURL.path()), subject, cc, bcc, body, attach;
    QStringList queries = QStringList::split('&', mailtoURL.query().mid(1));
diff -ur kdelibs.old/kfile/kopenwith.cpp kdelibs.new/kfile/kopenwith.cpp
--- kdelibs.old/kfile/kopenwith.cpp	Mon Jul 16 18:27:36 2001
+++ kdelibs.new/kfile/kopenwith.cpp	Sun Jan 27 22:11:44 2002
@@ -502,7 +502,7 @@
   if (terminal->isChecked()) {
     KSimpleConfig conf(QString::fromLatin1("konquerorrc"), true);
     conf.setGroup(QString::fromLatin1("Misc Defaults"));
-    m_command = conf.readEntry(QString::fromLatin1("Terminal"), QString::fromLatin1("konsole"));
+    m_command = conf.readEntry(QString::fromLatin1("Terminal"), QString::fromLatin1("x-terminal-emulator"));
 
     m_command += QString::fromLatin1(" -e ");
     m_command += edit->url();
diff -ur kdelibs.old/kio/kmimetype.cpp kdelibs.new/kio/kmimetype.cpp
--- kdelibs.old/kio/kmimetype.cpp	Thu Nov  8 14:50:39 2001
+++ kdelibs.new/kio/kmimetype.cpp	Sun Jan 27 22:11:44 2002
@@ -614,11 +614,11 @@
   if (s.substituteUid() && !user.isEmpty())
   {
     if (s.terminal())
-      cmd = QString("konsole %1 -e su %2 -c %3").arg(opts).arg(user).arg(cmd);
+      cmd = QString("x-terminal-emulator %1 -e su %2 -c %3").arg(opts).arg(user).arg(cmd);
     else
       cmd = QString("kdesu -u %1 -- %2").arg(user).arg(cmd);
   } else if (s.terminal())
-    cmd = QString("konsole %1 -e %2").arg(opts).arg(cmd);
+    cmd = QString("x-terminal-emulator %1 -e %2").arg(opts).arg(cmd);
   else
   {
     // No special options, do straight run
diff -ur kdelibs.old/kio/krun.cpp kdelibs.new/kio/krun.cpp
--- kdelibs.old/kio/krun.cpp	Sun Jul 29 08:30:10 2001
+++ kdelibs.new/kio/krun.cpp	Sun Jan 27 22:11:44 2002
@@ -113,12 +113,12 @@
   if (_service.substituteUid() && !user.isEmpty())
   {
     if (_service.terminal())
-      exec = QString("konsole %1 -e su %2 -c %3").arg(_service.terminalOptions()).arg(user).arg(exec);
+      exec = QString("x-terminal-emulator %1 -e su %2 -c %3").arg(_service.terminalOptions()).arg(user).arg(exec);
     else
       exec = QString("kdesu -u %1 -- %2").arg(user).arg(exec);
   } else if (_service.terminal())
      // Keep in mind that exec could include spaces. No quotes around %2, then.
-     exec = QString("konsole %1 -e %2").arg(_service.terminalOptions()).arg(exec);
+     exec = QString("x-terminal-emulator %1 -e %2").arg(_service.terminalOptions()).arg(exec);
   else
   {
   QString error;
diff -ur kdelibs.old/kio/rlogin.protocol kdelibs.new/kio/rlogin.protocol
--- kdelibs.old/kio/rlogin.protocol	Sat Jan 20 17:01:05 2001
+++ kdelibs.new/kio/rlogin.protocol	Sun Jan 27 22:11:44 2002
@@ -1,5 +1,5 @@
 [Protocol]
-exec=konsole -e rlogin `echo %u | sed -e 's,rlogin:/*,,'`
+exec=x-terminal-emulator -e rlogin `echo %u | sed -e 's,rlogin:/*,,'`
 protocol=rlogin
 input=none
 output=none
diff -ur kdelibs.old/kio/telnet.protocol kdelibs.new/kio/telnet.protocol
--- kdelibs.old/kio/telnet.protocol	Wed Apr 11 00:37:55 2001
+++ kdelibs.new/kio/telnet.protocol	Sun Jan 27 22:11:44 2002
@@ -1,5 +1,5 @@
 [Protocol]
-exec=konsole -e telnet `echo %u | sed -e 's,telnet:/*,,' | sed -e 's,:\([0-9]*\), \1,'`
+exec=x-terminal-emulator -e telnet `echo %u | sed -e 's,telnet:/*,,' | sed -e 's,:\([0-9]*\), \1,'`
 protocol=telnet
 input=none
 output=none
