diff -ur kdelibs.old/admin/acinclude.m4.in kdelibs.new/admin/acinclude.m4.in
--- kdelibs.old/admin/acinclude.m4.in	Mon Dec 17 14:46:09 2001
+++ kdelibs.new/admin/acinclude.m4.in	Sun Jan 27 22:18:44 2002
@@ -376,7 +376,7 @@
 if test "$1" = "default"; then
 
   if test -z "$kde_htmldir"; then
-    kde_htmldir='\${prefix}/share/doc/HTML'
+    kde_htmldir='\${prefix}/share/doc/kde/HTML'
   fi
   if test -z "$kde_appsdir"; then
     kde_appsdir='\${prefix}/share/applnk'
@@ -394,7 +394,7 @@
     kde_locale='\${prefix}/share/locale'
   fi
   if test -z "$kde_cgidir"; then
-    kde_cgidir='\${exec_prefix}/cgi-bin'
+    kde_cgidir='\${exec_prefix}/lib/cgi-bin'
   fi
   if test -z "$kde_confdir"; then
     kde_confdir='\${prefix}/share/config'
diff -ur kdelibs.old/arts/gmcop/Makefile.am kdelibs.new/arts/gmcop/Makefile.am
--- kdelibs.old/arts/gmcop/Makefile.am	Fri May 18 20:13:24 2001
+++ kdelibs.new/arts/gmcop/Makefile.am	Sun Jan 27 22:18:44 2002
@@ -1,8 +1,8 @@
 lib_LTLIBRARIES = libgmcop.la
 
-GLIB_PREFIX = /usr/local
+GLIB_PREFIX = /usr
 
-INCLUDES = -I$(top_srcdir)/arts/mcop $(GLIB_CFLAGS) $(all_includes)
+INCLUDES = -I$(top_srcdir)/arts/mcop -I/usr/include/glib-2.0/glib $(GLIB_CFLAGS) $(all_includes)
 
 # $(GLIB_DEPLIBS) ?
 libgmcop_la_SOURCES = giomanager.cc
diff -ur kdelibs.old/doc/api/configure.in.in kdelibs.new/doc/api/configure.in.in
--- kdelibs.old/doc/api/configure.in.in	Tue Jun 26 11:38:02 2001
+++ kdelibs.new/doc/api/configure.in.in	Sun Jan 27 22:18:44 2002
@@ -4,7 +4,7 @@
   kde_qtdir="$with_qt_dir"
 fi
 
-AC_FIND_FILE(qcolordialog.html, [ $kde_qtdir/doc/html $QTDIR/doc/html /usr/share/doc/packages/qt2/html /usr/lib/qt/doc /usr/doc/qt2/html /usr/doc/qt2], QTDOCDIR)
+AC_FIND_FILE(qcolordialog.html, [ $kde_qtdir/doc/html $QTDIR/doc/html /usr/share/doc/packages/qt2/html /usr/lib/qt/doc /usr/doc/qt2/html /usr/share/doc/qt-doc/doc/html], QTDOCDIR)
 AC_MSG_RESULT($QTDOCDIR)
 
 AC_SUBST(QTDOCDIR)
diff -ur kdelibs.old/kdecore/kstddirs.cpp kdelibs.new/kdecore/kstddirs.cpp
--- kdelibs.old/kdecore/kstddirs.cpp	Fri Sep 21 13:10:47 2001
+++ kdelibs.new/kdecore/kstddirs.cpp	Sun Jan 27 22:18:44 2002
@@ -692,7 +692,7 @@
     if (!strcmp(type, "data"))
 	return "share/apps/";
     if (!strcmp(type, "html"))
-	return "share/doc/HTML/";
+	return "share/doc/kde/HTML/";
     if (!strcmp(type, "icon"))
 	return "share/icons/";
     if (!strcmp(type, "config"))
@@ -712,7 +712,7 @@
     if (!strcmp(type, "mime"))
 	return "share/mimelnk/";
     if (!strcmp(type, "cgi"))
-	return "cgi-bin/";
+	return "lib/cgi-bin/";
     if (!strcmp(type, "wallpaper"))
 	return "share/wallpapers/";
     if (!strcmp(type, "templates"))
