diff -ur kdelibs.old/kdoctools/Makefile.am kdelibs.new/kdoctools/Makefile.am
--- kdelibs.old/kdoctools/Makefile.am	Thu Nov  8 06:25:21 2001
+++ kdelibs.new/kdoctools/Makefile.am	Mon Feb 18 22:19:09 2002
@@ -39,13 +39,12 @@
 	$(XGETTEXT) *.cpp -o $(podir)/kdebase/kio_help.pot
 
 install-data-local:
-	@(cd $(srcdir) && \
-	for subdir in customization docbook; do \
-	   for dir in `find $$subdir -type d ! -name CVS `; do \
-		$(mkinstalldirs) $(DESTDIR)$(kde_datadir)/ksgmltools2/$$dir ;\
-		for file in `ls -1 $$dir/*`; do \
-		    if test -f $$file; then \
-			$(INSTALL_DATA) $$file  $(DESTDIR)$(kde_datadir)/ksgmltools2/$$file; fi ;\
+	@(for subdir in customization docbook; do \
+	   for dir in `(cd $(srcdir)/$$subdir && find . -type d ! -name CVS )`; do \
+		$(mkinstalldirs) $(DESTDIR)$(kde_datadir)/ksgmltools2/$$subdir/$$dir ;\
+		for file in `(cd $(srcdir)/$$subdir && ls -1 $$dir/*)`; do \
+		    if test -f $(srcdir)/$$subdir/$$file; then \
+			$(INSTALL_DATA) $(srcdir)/$$subdir/$$file  $(DESTDIR)$(kde_datadir)/ksgmltools2/$$subdir/$$file; fi ;\
 	   	done ;\
 	   done ;\
 	done )
