--- kdelibs-2.1.0.orig/kssl/ksslinfodlg.cc
+++ kdelibs-2.1.0/kssl/ksslinfodlg.cc
@@ -67,7 +67,7 @@
         }
     } else {
         pixmap->setPixmap(BarIcon("unlock"));
-        info->setText(i18n("SSL support is not available in this build of KDE."));
+        info->setText(i18n("No SSL support!  Install kdebase-crypto."));
     }
     d->m_layout->addRowSpacing( 0, 50 ); // give minimum height to look better
 
