--- kdelibs-2.1.0.orig/ksgmltools/configure.in.in
+++ kdelibs-2.1.0/ksgmltools/configure.in.in
@@ -11,7 +11,7 @@
   IFS=$ac_save_IFS])
 
 dnl The order of the directories in this list matters
-dtd_paths="/usr/share/sgml/sgml-iso-entities-8879.1986 /usr/share/sgml/jade-1.2.1 /usr/share/sgml/docbook/sgml-dtd-3.1 /usr/share/sgml/docbook/dsssl-stylesheets-1.54 /usr/share/sgml /usr/local/share/sgml /etc /etc/sgml /usr/share/sgml /usr/share/sgml/docbk31 /usr/local/lib/sgml $USER_SGML_CAT_PATH"
+dtd_paths="/usr/share/sgml/sgml-iso-entities-8879.1986 /usr/share/sgml/jade-1.2.1 /usr/share/sgml/docbook/sgml-dtd-3.1 /usr/share/sgml/docbook/dsssl-stylesheets-1.54 /usr/share/sgml /usr/local/share/sgml /etc /etc/sgml /usr/share/sgml /usr/share/sgml/docbk31 /usr/local/lib/sgml $USER_SGML_CAT_PATH /usr/share/sgml-data"
 
 AC_DEFUN(KDE_FIND_CAT,
 [
@@ -28,7 +28,7 @@
 KDE_FIND_CAT(nwalsh-modular.cat)
 KDE_FIND_CAT(stylesheets.cat)
 KDE_FIND_CAT(dsssl.cat)
-KDE_FIND_CAT(sgml.catalog)
+KDE_FIND_CAT(sgml/catalog)
 dnl These three lines were added for the FreeBSD setup together with /usr/local/share/sgml
 KDE_FIND_CAT(docbook/3.1/catalog)
 KDE_FIND_CAT(jade/catalog)
--- kdelibs-2.1.0.orig/kssl/ksslinfodlg.cc
+++ kdelibs-2.1.0/kssl/ksslinfodlg.cc
@@ -67,7 +67,7 @@
         }
     } else {
         pixmap->setPixmap(BarIcon("unlock"));
-        info->setText(i18n("SSL support is not available in this build of KDE."));
+        info->setText(i18n("No SSL support!  Install kdebase-crypto."));
     }
     d->m_layout->addRowSpacing( 0, 50 ); // give minimum height to look better
 
