--- kdelibs-2.1-final.orig/kssl/ksslinfodlg.cc
+++ kdelibs-2.1-final/kssl/ksslinfodlg.cc
@@ -67,7 +67,7 @@
         }
     } else {
         pixmap->setPixmap(BarIcon("unlock"));
-        info->setText(i18n("SSL support is not available in this build of KDE."));
+        info->setText(i18n("No SSL support!  Install kdebase-crypto."));
     }
     d->m_layout->addRowSpacing( 0, 50 ); // give minimum height to look better
 
--- kdelibs-2.1-final.orig/admin/debianrules
+++ kdelibs-2.1-final/admin/debianrules
@@ -74,7 +74,7 @@
   print STDOUT "export KDB2HTML=$KDB2HTML\n";
   print STDOUT "export MCOPIDL=$MCOPIDL\n";
   print STDOUT "export ARTSCCONFIG=$ARTSCCONFIG\n";
-  print STDOUT "configkde=--disable-rpath --prefix=\$(kde_prefix) --libexecdir=\$(kde_bindir) --with-qt-includes=/usr/include/qt --mandir=\$(mandir) --infodir=\$(infodir)\n";
+  print STDOUT "configkde=--disable-debug --disable-rpath --prefix=\$(kde_prefix) --libexecdir=\$(kde_bindir) --with-qt-includes=/usr/include/qt --mandir=\$(mandir) --infodir=\$(infodir)\n";
   print STDOUT "configkdevelop=--enable-docbase --enable-kdoc2 --disable-rpath --libdir=\$(kde_libdir) --includedir=\$(kde_includedir) --with-qt-includes=/usr/include/qt --mandir=\$(mandir) --with-kdelibsdoc-dir=/usr/share/doc/kdelibs3-doc/html\n";
   print STDOUT "configkdepim=--with-extra-includes=/usr/include/libpisock --enable-shared --disable-rpath --prefix=\$(kde_prefix) --libexecdir=\$(kde_bindir) --with-qt-includes=/usr/include/qt --mandir=\$(mandir) --infodir=\$(infodir)\n";
   exit
